components/calendar/src/ethiopic.rs

```diff
@@ -256,7 +256,7 @@ impl DateTime<Ethiopic> {
     /// ```rust
     /// use icu::calendar::{types::IsoHour, types::IsoMinute, types::IsoSecond, DateTime};
     ///
-    /// let datetime_ethiopic = DateTime::new_ethiopic_datetime(2014, 8, 25, 13, 1, 0, 0)
+    /// let datetime_ethiopic = DateTime::new_ethiopic_datetime(2014, 8, 25, 13, 1, 0)
```

Comment:
Issue: Why are there changes in these files? Maybe a merge mistake? Please revert.

Reply:
This is from discussion above https://github.com/unicode-org/icu4x/pull/1887#issuecomment-1130454138

Reply:
Yeah, I asked for these changes and felt they were minor enough to include in this PR

Reply:
I don't get it; isn't it a regression to remove the fractional seconds argument from this function?

Reply:
The fractional seconds argument was only there on two of the calendars, so it was not uniform, it was broken either way. Fractional seconds can always be set manually (the fields are public); but I don't think it's that important to have them on the main constructor, so I opted to remove them. If we want them we should add them uniformly to all the constructors and I'd prefer that happen separately.

Reply:
I would have probably gone the other way (add fractional seconds where it was missing), but OK.